src/components/map/GeographySelect.tsx

```diff
@@ -26,17 +26,18 @@ const GeographySelect = ({ title, list, keyField, keyFieldDisplay, filterType, h
       setSuggestList([]);
       return;
     }
-    let filteredList = [];
+    let filteredList: Record<string, unknown>[] = [];
     filteredList = Object.values(list).filter((item) => {
-      return item[keyFieldDisplay ?? keyField].toLowerCase().indexOf(input.toLowerCase()) > -1;
+      const record = item as Record<string, string>;
```

Comment:
Thought: Can't `list` be more correctly typed higher up to prevent the need for this coercion here?